src/status_im/ui/screens/pairing/views.cljs

```diff
@@ -74,13 +69,10 @@
      [react/view
       [react/text (i18n/label :t/pair-this-device-description)]]]]])
 
-(defn your-device [{:keys [installation-id name device-type]}]
+(defn your-device [{:keys [installation-id name]}]
   [react/view {:style styles/installation-item}
    [react/view {:style (styles/pairing-button true)}
-    [icons/icon  (if (= "desktop"
-                        device-type)
-                   :main-icons/desktop
-                   :main-icons/mobile)
+    [icons/icon :main-icons/mobile
```

Comment:
Not so fast :) 
this one it just means you are paired with a desktop machine, I think we should keep it, although desktop has no device pairing yet.

Reply:
Addressed by Andrey.